pvlib/modelchain.py

```diff
@@ -746,7 +720,7 @@ def effective_irradiance_model(self):
             fd*self.total_irrad['poa_diffuse'])
         return self
 
-    def complete_irradiance(self, times=None, weather=None):
+    def complete_irradiance(self, weather, times=None):
```

Comment:
Does the change in argument order need to be mentioned in whatsnew?  Same comment on `prepare_inputs`, `run_model`.

Reply:
Doesn't hurt, but I think it's considered bad form to rely on the position of keyword arguments in code that you don't control.